### PR TITLE
Fix port script

### LINKED ISSUE #2565: Postgres migration script appears to be broken (again)
### Description

```
2017-10-22 02:08:45,904 - synapse_port_db - 546 - ERROR -
Traceback (most recent call last):
  File "/home/travis/.synapse/bin/synapse_port_db", line 539, in run
    consumeErrors=True,
FirstError: FirstError[#9, [Failure instance: Traceback: <type 'exceptions.AttributeError'>: type object 'Store' has no attribute '_simple_update_txn'
/home/travis/.synapse/local/lib/python2.7/site-packages/twisted/internet/defer.py:500:errback
/home/travis/.synapse/local/lib/python2.7/site-packages/twisted/internet/defer.py:567:_startRunCallbacks
/home/travis/.synapse/local/lib/python2.7/site-packages/twisted/internet/defer.py:653:_runCallbacks
/home/travis/.synapse/local/lib/python2.7/site-packages/twisted/internet/defer.py:1442:gotResult
--- <exception caught here> ---
/home/travis/.synapse/local/lib/python2.7/site-packages/twisted/internet/defer.py:1384:_inlineCallbacks
/home/travis/.synapse/local/lib/python2.7/site-packages/twisted/python/failure.py:408:throwExceptionIntoGenerator
/home/travis/.synapse/bin/synapse_port_db:338:handle_table
/home/travis/.synapse/local/lib/python2.7/site-packages/twisted/python/threadpool.py:250:inContext
/home/travis/.synapse/local/lib/python2.7/site-packages/twisted/python/threadpool.py:266:<lambda>
/home/travis/.synapse/local/lib/python2.7/site-packages/twisted/python/context.py:122:callWithContext
/home/travis/.synapse/local/lib/python2.7/site-packages/twisted/python/context.py:85:callWithContext
/home/travis/.synapse/local/lib/python2.7/site-packages/twisted/enterprise/adbapi.py:307:_runWithConnection
/home/travis/.synapse/local/lib/python2.7/site-packages/twisted/enterprise/adbapi.py:298:_runWithConnection
/home/travis/.synapse/bin/synapse_port_db:126:r
/home/travis/.synapse/bin/synapse_port_db:334:insert
/home/travis/.synapse/local/lib/python2.7/site-packages/synapse/storage/_base.py:800:_simple_update_one_txn
]]
```

This is on a dev environment, which might be part of the problem. Server is based off of the develop branch.

### Version information

- **Homeserver**: self-hosted dev synapse
- **Version**:        0.24.0-rc1 (develop branch)
- **Install method**: pip
- **Platform**:    Ubuntu 16.04 VM

### PR DESCRIPTION
**Pull request by erikjohnston**
We changed _simple_update_one_txn to use _simple_update_txn but didn't
yank it out in the port script.

Fixes #2565

### CODE REVIEW

File: scripts/synapse_port_db [hunk 1]
```diff
@@ -112,6 +112,7 @@ class Store(object):
 
     _simple_update_one = SQLBaseStore.__dict__["_simple_update_one"]
```
Comment (richvdh) [inline scripts/synapse_port_db]: this is redundant, fwiw

File: scripts/synapse_port_db [hunk 2]
```diff
@@ -112,6 +112,7 @@ class Store(object):
...
     _simple_update_one_txn = SQLBaseStore.__dict__["_simple_update_one_txn"]
```
Comment (richvdh) [inline scripts/synapse_port_db]: likewise

Comment (erikjohnston) [reply scripts/synapse_port_db]: It gets used as well in the script?

Comment (richvdh) [reply scripts/synapse_port_db]: I'm a crank. ignore me.

File: scripts/synapse_port_db (hunk 1 above)
Comment (richvdh) [reply scripts/synapse_port_db]: I'm a crank

**Pull request merged**